test/cppsuite/test_harness/core/throttle.h

```diff
@@ -30,34 +30,46 @@
 #define THROTTLE_H
 
 #include <thread>
+#include <string>
 
 #include "configuration.h"
 
 namespace test_harness {
 class throttle {
     public:
-    throttle(const int64_t op_count, const char interval)
+    explicit throttle(std::string throttle_rate)
```

Comment:
You're not modifying `throttle_rate` so may as well take `const std::string&` instead of `std::string`.

Reply:
701752